content/en/docs/armory-platform-matrix.md

```diff
@@ -195,7 +175,7 @@ The following table lists the supported authorization methods:
 | OAuth 2.0/OIDC        | All supported versions |                                                                                      |
```

Comment:
I'm a TOUCH hesitant on this.  Maybe add a note "Requires provider to include groups in claims or be a supported third party integration."

Reply:
Looks like Github is the only provider in gate for oauth. Should we specifically call that out?